tests/testsrc/test262.properties

```diff
@@ -2555,17 +2947,69 @@ built-ins/TypedArrayConstructors 198/736 (26.9%)
     internals/Set/resized-out-of-bounds-to-in-bounds-index.js {unsupported: [resizable-arraybuffer]}
     internals/Set/tonumber-value-throws.js
 
-built-ins/WeakMap 11/119 (9.24%)
-    prototype/getOrInsert/adds-object-element.js
-    prototype/getOrInsert/adds-symbol-element.js
-    prototype/getOrInsert/getOrInsert.js
-    prototype/getOrInsert/length.js
-    prototype/getOrInsert/name.js
-    prototype/getOrInsert/not-a-constructor.js
-    prototype/getOrInsert/returns-value-if-key-is-not-present-object-key.js
-    prototype/getOrInsert/returns-value-if-key-is-not-present-symbol-key.js
-    prototype/getOrInsert/returns-value-if-key-is-present-object-key.js
-    prototype/getOrInsert/returns-value-if-key-is-present-symbol-key.js
```

Comment:
I'm not 100% sure, if I got it, but a removed line means, that the test is no longer excluded and will pass now?

Can you explain me, why an update of the test262 with nearly no other code changes lets them pass now?
Am I missing something or was the file not up-to-date on master?

Reply:
> I'm not 100% sure, if I got it, but a removed line means, that the test is no longer excluded and will pass now?

excluded things are marked with a ~ in front, all explicit listed tests are executed and failing. Will have a look at these - but keep in mind i only have regenerated the file (and added some more folder)